packages/fluentui/react-northstar/src/components/Animation/Animation.tsx

```diff
@@ -14,14 +11,15 @@ import * as React from 'react';
 import { ThemeContext } from 'react-fela';
 import { Transition } from 'react-transition-group';
 
-import { childrenExist, StyledComponentProps, commonPropTypes, ChildrenComponentProps } from '../../utils';
+import { childrenExist, commonPropTypes, ChildrenComponentProps } from '../../utils';
 import { ComponentEventHandler, ProviderContextPrepared } from '../../types';
+import useAnimationStyles from './useAnimationStyles';
+
+export { animationClassName } from './useAnimationStyles';
 
 export type AnimationChildrenProp = (props: { classes: string }) => React.ReactNode;
 
-export interface AnimationProps
-  extends StyledComponentProps,
-    ChildrenComponentProps<AnimationChildrenProp | React.ReactChild> {
+export interface AnimationProps extends ChildrenComponentProps<AnimationChildrenProp | React.ReactChild> {
```

Comment:
We were laying that we support styles and variables, so I removed those props. This will allow us for better performance and at least caching always when it is enabled.